v19.1/managed-connect-to-your-cluster.md

```diff
@@ -7,50 +7,64 @@ build_for: [managed]
 
 Once your Managed CockroachDB cluster is available and you've [set your password](managed-sign-up-for-a-cluster.html#set-your-password) and [signed in](managed-sign-up-for-a-cluster.html#sign-in), you can start interacting with your cluster.
 
-## Before you begin
+## Step 1. Authorize your network
 
-1. On the machine from which you want to connect to the cluster, create a `certs` directory. The `ca.crt` file that you download later will go here.
+Managed CockroachDB requires you to authorize the networks that can access the cluster. This helps prevent denial-of-service and brute force password attacks.
 
-    Any client connecting to the cluster will need access to the `ca.crt` file.
+You need to authorize your application server’s network and your local machine’s network. If you change your location, you need to authorize the new location’s network, else the connection from that network will be rejected.
```

Comment:
You only need to  authorize your local machine's network if you intend to make SQL queries from it. For most prod deployments, you probably don't want to authorize your local machine's network.